tests/test_actions.py

```diff
@@ -246,11 +262,9 @@ def test_key_value_set(self):
         assert mn.key == test_key
 
     def test_answers_question(self, Tester):
-        MockQuestion = mock.Mock()
-
+        MockQuestion = mock.create_autospec(Question)
```

Comment:
Ooo, another neat side-benefit of your `@runtime_checkable` additions!

Reply:
Indeed, but I forgot to add the decorator to the temporary `Describable` protocol in the `unittest_protocols.py` module. It's unclear to me what the ramifications of this are.